app/assets/stylesheets/themes/default/_participant.scss

```diff
@@ -0,0 +1,146 @@
+.participant-content {
+  background: white;
+  padding: 20px;
+  margin-bottom: 20px;
+
+  &__title {
+    color: $base-color;
+    font-size: $default-font-size;
+    margin: 0;
+  }
+
+  &__description {
+    margin: 1em 0 0 0;
+  }
+}
+
+.participant-titles {
+  border-top: 2px solid rgba($base-color, 0.2);
+  margin-top: 20px;
+  padding-top: 10px;
+  position: relative;
+
+  li {
+    margin-bottom: 10px;
+    margin-left: 1.5em;
+
+    &:before {
+      content: ' ';
+      border: transparent solid 0.3em;
+      border-left-color: $base-color;
+      position: absolute;
+      height: 0;
+      width: 0;
+      left: 0.5em;
+      margin: 0px;
+      margin-top: 0.4em;
+      padding: 0px;
+    }
+
+    strong {
+      font-size: $default-font-size - 2;
+      padding: 2px 10px;
+      color: white;
+      background-color: $base-color;
+      border-radius: $input-border-radius;
+    }
+  }
+
+  @include media($smaller-tablet) {
+    li {
+      font-size: $default-font-size - 2;
+    }
+  }
+}
+
+@include media($larger-tablet) {
```

Comment:
Sorry, I don't know what you pointed out. Need to flip class name and media query? Which is consistent, `_post.scss` or `_serial.scss`?